<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>975</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonCheck">
     <property name="text">
      <string>Check an archive for possible erroneous entries</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxSimilarLocators">
     <property name="text">
      <string>Show similar locators</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonHelp">
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonExit">
     <property name="text">
      <string>Close the Archive Checker Application</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>929</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
